### PR TITLE
feat: customize browser tab title

### PR DESCRIPTION
**Pull request by sceccotti89**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fastify/swagger-ui",
-  "version": "1.6.0",
+  "version": "1.7.0",
```
Comment (mcollina) [inline package.json:3]: Please revert this, I will bump it on release

**Review by mcollina — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (sceccotti89) [reply package.json:3]: This is from @Eomm but I can revert it.

File: types/index.d.ts [hunk 2]
```diff
@@ -60,9 +60,10 @@ declare namespace fastifySwaggerUi {
   }
 
   type FastifySwaggerUiTheme = {
+    title?: string;
     css?: { filename: string; content: string; }[];
     js?: { filename: string; content: string; }[];
-    favicon: { filename: string; rel: string; type: string; sizes: string; content: string | Buffer; }[];
+    favicon?: { filename: string; rel: string; type: string; sizes: string; content: string | Buffer; }[];
```
[the remark this answers is not in the document]
Comment (sceccotti89) [reply types/index.d.ts:66]: I've added an additional test to cover the theme's title. Not sure how to assert the optional favicon.

File: package.json (hunk 1 above)
Comment (mcollina) [reply package.json]: Please do, thx

Comment (mcollina) [reply package.json]: You might need to rebase your changes

**Comment by sceccotti89** [comment]
@mcollina PR updated.

**Review by Eomm — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**